pkg/server/server.go

```diff
@@ -1,5 +1,6 @@
 package server
 
-//go:generate sh -c "protoc -I../../thirdparty/proto/api-common-protos -I ../.. ../../pkg/server/proto/server.proto --go_out=plugins=grpc:../.. --go-json_out=../.."
+//go:generate sh -c "protoc -I../../thirdparty/proto/api-common-protos -I ../.. ../../pkg/server/proto/server.proto --go_out=plugins=grpc:../.. --go-json_out=../.. --swagger_out=../.."
+//go:generate mv ./proto/server.swagger.json ./gen/
```

Comment:
Are there any extra tools that should go in `make tools` for generating swagger?

Reply:
Yes! Done